src/config/consts.js

```diff
@@ -53,6 +53,9 @@ export const consts = {
     website: '(Webseite)',
     webViewHint: '(Öffnet Webseite in der aktuellen App)'
   },
+
+  EMAIL_REGEX: /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/,
```

Comment:
this seems not to be enough:

![Bildschirmfoto 2022-05-20 um 19 08 27](https://user-images.githubusercontent.com/1942953/169578436-88af2564-4289-4230-82bd-e5ae4302afca.png)

`.de` is missing but it gets valid

Reply:
then the same problem exists in Consul

Reply:
true. with this PR, both will use the same regex. i will investigate further on monday.

Reply:
98e8b40bbee458039eb8e6a3cb373a7fc08f28f0